From e015d6683e0a45b8cd1c430f4a07df5fe97e6162 Mon Sep 17 00:00:00 2001
From: Ariel Aviad <aaviad@habana.ai>
Date: Sun, 02 Mar 2025 13:43:32 +0200
Subject: [PATCH] [FSW-6867] fs1: palad: replace pldm efuse macros to palad

In specs we duplicate pdlm efuse define as palad name
this fix in using the palad instead of pldm
preparing to remove the pldm defines
as part of renaming pldm to palad

Change-Id: I75780a7d191ebbae3d06abf03cdc8a000571f3e2
Signed-off-by: Ariel Aviad <aaviad@habana.ai>
---

diff --git a/arch/fs1/arc/palad.c b/arch/fs1/arc/palad.c
index 84f82ac..29da419 100644
--- a/arch/fs1/arc/palad.c
+++ b/arch/fs1/arc/palad.c
@@ -122,7 +122,7 @@
 	u8 num_vf = 0;
 	u32 val;
 
-	hl_fuse_read(mmIO_0_CFG_HALF_0_PSOC_PSOC_CFG_EFUSE_BASE, IO_EFUSE_PLDM_CFG_0_OFFSET,
+	hl_fuse_read(mmIO_0_CFG_HALF_0_PSOC_PSOC_CFG_EFUSE_BASE, IO_EFUSE_PALAD_CFG_0_OFFSET,
 		     &pci_efuse_cfg);
 	preboot_puts_hex("efuse pci cfg = ", pci_efuse_cfg, false);
 
@@ -359,7 +359,7 @@
 	hw_mem_copy(IO0_MISC_ICCM_FULL_BASE, PREBOOT_TEXT_BASE_FULL, IO_MISC_ICCM_SIZE, false,
 		     false);
 
-	err = preboot_fuse_read(IO_EFUSE_PLDM_CFG_0_OFFSET, &val);
+	err = preboot_fuse_read(IO_EFUSE_PALAD_CFG_0_OFFSET, &val);
 	if (err)
 		val = BLOADER_COPY_LARGE_APP;
 
@@ -377,7 +377,7 @@
 	int err;
 	u32 val;
 
-	err = preboot_fuse_read(IO_EFUSE_PLDM_CFG_0_OFFSET, &val);
+	err = preboot_fuse_read(IO_EFUSE_PALAD_CFG_0_OFFSET, &val);
 	if (err)
 		val = BLOADER_COPY_BOOTINFO;
 
